test/Microsoft.VisualStudio.Web.CodeGeneration.Templating.Test/RazorTemplatingTests.cs

```diff
@@ -76,7 +76,7 @@ private static ProjectContext CreateProjectContext(string projectPath)
             var framework = NuGet.Frameworks.FrameworkConstants.CommonFrameworks.Net451.GetShortFolderName();
 #else
             projectPath = projectPath ?? Directory.GetCurrentDirectory();
-            var framework = NuGet.Frameworks.FrameworkConstants.CommonFrameworks.NetCoreApp10.GetShortFolderName();
+            var framework = "netcoreapp1.1";
```

Comment:
Spoke to Pranav, tools cannot update to target to netcoreapp1.1, so will remove that.